recipes/LibriTTS/vocoder/hifigan/train.py

```diff
@@ -363,21 +363,21 @@ def audio_pipeline(wav, segment):
         overrides=overrides,
     )
 
-    sys.path.append("../../")
```

Comment:
there should be a symlink in that folder now, so that scripts which are in `../...` can be loaded directly. This line will prohibit recipe execution from any other place, sine that `../..` relative path will be pointing completely elsewhere